examples/animation/double_pendulum.py

```diff
@@ -64,8 +63,13 @@ def derivs(state, t):
 # initial state
 state = np.radians([th1, w1, th2, w2])
 
-# integrate your ODE using scipy.integrate.
-y = integrate.odeint(derivs, state, t)
+# integrate the ODE using Euler's method
```

Comment:
This makes the example longer and IMHO a little harder to understand. Does this outweigh the benefit of not depending on scipy in this example?

Reply:
The only other use of scipy in the docs can also easily be removed (I have another patch ready for that) so we could entirely drop the dependency on scipy for building the docs, which may or may not be a useful thing (e.g., with the yearly release cycle of cpython, I fear that we're always going to have a small period between having scipy wheels available after the new cpython release, and scipy is significantly harder to build from source (e.g., due to fortran sources) than other dependencies, even numpy).

Reply:
I think dropping SciPy would be pretty helpful if we don't need it.  I think its a pretty heavy dependency to carry around for a couple of examples.

Reply:
We have oscillated back and forth on having a scipy documentation dependencies a couple of times.  I do not have a strong preference either way.